pkg/cloud/cloud.go

```diff
@@ -334,6 +393,15 @@ func isDriverBootedInECS() bool {
 	return ecsContainerMetadataUri != ""
 }
 
+func isAccessPointAlreadyExists(err error) bool {
```

Comment:
Redundant function? Not seeing any usage

Reply:
Removed this function